2020-10-16 12:53:15,743 ERROR 15984 nodejs.ClusterClientNoResponseError: client no response in 139131ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (C:\Users\12552\Desktop\dada\serve\node_modules\_cluster-client@3.0.1@cluster-client\lib\leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 15984
hostname: DESKTOP-K5ULKD4

2020-10-16 12:53:15,742 ERROR 15356 nodejs.ECONNABORTEDError: write ECONNABORTED (address: 127.0.0.1:60763)
    at afterWriteDispatched (internal/stream_base_commons.js:154:25)
    at writeGeneric (internal/stream_base_commons.js:145:3)
    at Socket._writeGeneric (net.js:786:11)
    at Socket._write (net.js:798:8)
    at doWrite (_stream_writable.js:403:12)
    at writeOrBuffer (_stream_writable.js:387:5)
    at Socket.Writable.write (_stream_writable.js:318:11)
    at Follower.sendHeartBeat (C:\Users\12552\Desktop\dada\serve\node_modules\_tcp-base@3.1.0@tcp-base\lib\base.js:105:18)
    at Timeout._onTimeout (C:\Users\12552\Desktop\dada\serve\node_modules\_tcp-base@3.1.0@tcp-base\lib\base.js:437:12)
    at listOnTimeout (internal/timers.js:549:17)
errno: "ECONNABORTED"
code: "ECONNABORTED"
syscall: "write"
name: "ECONNABORTEDError"
pid: 15356
hostname: DESKTOP-K5ULKD4

2020-10-16 15:35:26,779 ERROR 14196 nodejs.ServerNoResponseError: server 127.0.0.1:60763 no response in 40000ms, maybe the socket is end on the other side. (address: 127.0.0.1:60763)
    at Timeout._onTimeout (C:\Users\12552\Desktop\dada\serve\node_modules\_tcp-base@3.1.0@tcp-base\lib\base.js:427:21)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ServerNoResponseError"
pid: 14196
hostname: DESKTOP-K5ULKD4

2020-10-16 15:49:51,724 ERROR 15984 nodejs.ClusterClientNoResponseError: client no response in 924947ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (C:\Users\12552\Desktop\dada\serve\node_modules\_cluster-client@3.0.1@cluster-client\lib\leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 15984
hostname: DESKTOP-K5ULKD4

2020-10-16 15:49:58,334 ERROR 8692 [-/192.168.3.150/-/16ms GET /addorder?obj=%7B%22price%22:%22100~500%E5%85%83%22,%22quality%22:5,%22info%22:%22adasdadasdasdasda%22,%22qianshou%22:true,%22fahuoTime%22:%222020-10-21+15:49:23%22,%22address1%22:%22fgsg%22,%22address2%22:%22a%22,%22phone1%22:%2218212418574%22,%22phone2%22:%2218212418574%22,%22menpai1%22:%22d%22,%22menpai2%22:%2217%22,%22sid%22:1,%22uid%22:9%7D] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'order ((quality,uid,qianshou,price,time,info,sid)
         values ("undefined","' at line 1
         values ("undefined","' at line 1
    at Query.Sequence._packetToError (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Pool.js:199:23)
    at C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\12552\Desktop\dada\serve\node_modules\_ali-rds@3.4.0@ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (C:\Users\12552\Desktop\dada\serve\node_modules\_ali-rds@3.4.0@ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\12552\Desktop\dada\serve\node_modules\_co@4.6.0@co\index.js:65:19)
    at C:\Users\12552\Desktop\dada\serve\node_modules\_co@4.6.0@co\index.js:54:5
    sql: insert into order ((quality,uid,qianshou,price,time,info,sid)
         values ("undefined","undefined","undefined","undefined","undefined",
         "undefined")
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'order ((quality,uid,qianshou,price,time,info,sid)\n         values (\"undefined\",\"' at line 1"
sqlState: "42000"
index: 0
sql: "insert into order ((quality,uid,qianshou,price,time,info,sid)\n         values (\"undefined\",\"undefined\",\"undefined\",\"undefined\",\"undefined\",\n         \"undefined\")"
headers: {"Access-Control-Allow-Origin":"http://192.168.3.150:3000","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_PARSE_ERRORError"
pid: 8692
hostname: DESKTOP-K5ULKD4

2020-10-16 15:51:27,474 ERROR 17280 [-/192.168.3.150/-/16ms GET /addorder?obj=%7B%22price%22:%22100~500%E5%85%83%22,%22quality%22:5,%22info%22:%22%E6%B5%8B%E8%AF%95%E6%95%B0%E6%8D%AE%22,%22qianshou%22:true,%22fahuoTime%22:%222020-10-21+15:49:23%22,%22address1%22:%22fgsg%22,%22address2%22:%22a%22,%22phone1%22:%2218212418574%22,%22phone2%22:%2218212418574%22,%22menpai1%22:%22d%22,%22menpai2%22:%2217%22,%22sid%22:1,%22uid%22:9%7D] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'order ((quality,uid,qianshou,price,time,info,sid)
         values ("undefined","' at line 1
         values ("undefined","' at line 1
    at Query.Sequence._packetToError (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Pool.js:199:23)
    at C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\12552\Desktop\dada\serve\node_modules\_ali-rds@3.4.0@ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (C:\Users\12552\Desktop\dada\serve\node_modules\_ali-rds@3.4.0@ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\12552\Desktop\dada\serve\node_modules\_co@4.6.0@co\index.js:65:19)
    at C:\Users\12552\Desktop\dada\serve\node_modules\_co@4.6.0@co\index.js:54:5
    sql: insert into order ((quality,uid,qianshou,price,time,info,sid)
         values ("undefined","undefined","undefined","undefined","undefined",
         "undefined")
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'order ((quality,uid,qianshou,price,time,info,sid)\n         values (\"undefined\",\"' at line 1"
sqlState: "42000"
index: 0
sql: "insert into order ((quality,uid,qianshou,price,time,info,sid)\n         values (\"undefined\",\"undefined\",\"undefined\",\"undefined\",\"undefined\",\n         \"undefined\")"
headers: {"Access-Control-Allow-Origin":"http://192.168.3.150:3000","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_PARSE_ERRORError"
pid: 17280
hostname: DESKTOP-K5ULKD4

2020-10-16 15:54:24,829 ERROR 17280 [-/192.168.3.150/-/9207ms GET /addorder?obj=%7B%22price%22:%22100~500%E5%85%83%22,%22quality%22:5,%22info%22:%22%E6%B5%8B%E8%AF%95%E6%95%B0%E6%8D%AE%22,%22qianshou%22:true,%22fahuoTime%22:%222020-10-21+15:49:23%22,%22address1%22:%22fgsg%22,%22address2%22:%22a%22,%22phone1%22:%2218212418574%22,%22phone2%22:%2218212418574%22,%22menpai1%22:%22d%22,%22menpai2%22:%2217%22,%22sid%22:1,%22uid%22:9%7D] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'order ((quality,uid,qianshou,price,time,info,sid)
         values ("undefined","' at line 1
         values ("undefined","' at line 1
    at Query.Sequence._packetToError (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Pool.js:199:23)
    at C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\12552\Desktop\dada\serve\node_modules\_ali-rds@3.4.0@ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (C:\Users\12552\Desktop\dada\serve\node_modules\_ali-rds@3.4.0@ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\12552\Desktop\dada\serve\node_modules\_co@4.6.0@co\index.js:65:19)
    at C:\Users\12552\Desktop\dada\serve\node_modules\_co@4.6.0@co\index.js:54:5
    sql: insert into order ((quality,uid,qianshou,price,time,info,sid)
         values ("undefined","undefined","undefined","undefined","undefined",
         "undefined")
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'order ((quality,uid,qianshou,price,time,info,sid)\n         values (\"undefined\",\"' at line 1"
sqlState: "42000"
index: 0
sql: "insert into order ((quality,uid,qianshou,price,time,info,sid)\n         values (\"undefined\",\"undefined\",\"undefined\",\"undefined\",\"undefined\",\n         \"undefined\")"
headers: {"Access-Control-Allow-Origin":"http://192.168.3.150:3000","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_PARSE_ERRORError"
pid: 17280
hostname: DESKTOP-K5ULKD4

2020-10-16 15:54:33,288 ERROR 13192 [-/192.168.3.150/-/15ms GET /addorder?obj=%7B%22price%22:%22100~500%E5%85%83%22,%22quality%22:5,%22info%22:%22%E6%B5%8B%E8%AF%95%E6%95%B0%E6%8D%AE%22,%22qianshou%22:true,%22fahuoTime%22:%222020-10-21+15:49:23%22,%22address1%22:%22fgsg%22,%22address2%22:%22a%22,%22phone1%22:%2218212418574%22,%22phone2%22:%2218212418574%22,%22menpai1%22:%22d%22,%22menpai2%22:%2217%22,%22sid%22:1,%22uid%22:9%7D] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'order ((quality,uid,qianshou,price,time,info,sid)
         values ("undefined","' at line 1
         values ("undefined","' at line 1
    at Query.Sequence._packetToError (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Pool.js:199:23)
    at C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\12552\Desktop\dada\serve\node_modules\_ali-rds@3.4.0@ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (C:\Users\12552\Desktop\dada\serve\node_modules\_ali-rds@3.4.0@ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\12552\Desktop\dada\serve\node_modules\_co@4.6.0@co\index.js:65:19)
    at C:\Users\12552\Desktop\dada\serve\node_modules\_co@4.6.0@co\index.js:54:5
    sql: insert into order ((quality,uid,qianshou,price,time,info,sid)
         values ("undefined","undefined","undefined","undefined","undefined",
         "undefined")
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'order ((quality,uid,qianshou,price,time,info,sid)\n         values (\"undefined\",\"' at line 1"
sqlState: "42000"
index: 0
sql: "insert into order ((quality,uid,qianshou,price,time,info,sid)\n         values (\"undefined\",\"undefined\",\"undefined\",\"undefined\",\"undefined\",\n         \"undefined\")"
headers: {"Access-Control-Allow-Origin":"http://192.168.3.150:3000","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_PARSE_ERRORError"
pid: 13192
hostname: DESKTOP-K5ULKD4

2020-10-16 15:56:47,099 ERROR 11896 [-/192.168.3.150/-/15ms GET /addorder?obj=%7B%22price%22:%22100~500%E5%85%83%22,%22quality%22:5,%22info%22:%22%E6%B5%8B%E8%AF%95%E6%95%B0%E6%8D%AE%22,%22qianshou%22:true,%22fahuoTime%22:%222020-10-21+15:49:23%22,%22address1%22:%22fgsg%22,%22address2%22:%22a%22,%22phone1%22:%2218212418574%22,%22phone2%22:%2218212418574%22,%22menpai1%22:%22d%22,%22menpai2%22:%2217%22,%22sid%22:1,%22uid%22:9%7D] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''order' ((quality,uid,qianshou,price,time,info,sid)
         values ("undefined"' at line 1
         values ("undefined"' at line 1
    at Query.Sequence._packetToError (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Pool.js:199:23)
    at C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\12552\Desktop\dada\serve\node_modules\_ali-rds@3.4.0@ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (C:\Users\12552\Desktop\dada\serve\node_modules\_ali-rds@3.4.0@ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\12552\Desktop\dada\serve\node_modules\_co@4.6.0@co\index.js:65:19)
    at C:\Users\12552\Desktop\dada\serve\node_modules\_co@4.6.0@co\index.js:54:5
    sql: insert into 'order' ((quality,uid,qianshou,price,time,info,sid)
         values ("undefined","undefined","undefined","undefined","undefined",
         "undefined")
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''order' ((quality,uid,qianshou,price,time,info,sid)\n         values (\"undefined\"' at line 1"
sqlState: "42000"
index: 0
sql: "insert into 'order' ((quality,uid,qianshou,price,time,info,sid)\n         values (\"undefined\",\"undefined\",\"undefined\",\"undefined\",\"undefined\",\n         \"undefined\")"
headers: {"Access-Control-Allow-Origin":"http://192.168.3.150:3000","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_PARSE_ERRORError"
pid: 11896
hostname: DESKTOP-K5ULKD4

2020-10-16 15:58:00,972 ERROR 7944 [-/192.168.3.150/-/14ms GET /addorder?obj=%7B%22price%22:%22100~500%E5%85%83%22,%22quality%22:5,%22info%22:%22%E6%B5%8B%E8%AF%95%E6%95%B0%E6%8D%AE%22,%22qianshou%22:true,%22fahuoTime%22:%222020-10-21+15:49:23%22,%22address1%22:%22fgsg%22,%22address2%22:%22a%22,%22phone1%22:%2218212418574%22,%22phone2%22:%2218212418574%22,%22menpai1%22:%22d%22,%22menpai2%22:%2217%22,%22sid%22:1,%22uid%22:9%7D] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''order' (quality,uid,qianshou,price,time,info,sid)
         values ("undefined",' at line 1
         values ("undefined",' at line 1
    at Query.Sequence._packetToError (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Pool.js:199:23)
    at C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\12552\Desktop\dada\serve\node_modules\_ali-rds@3.4.0@ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (C:\Users\12552\Desktop\dada\serve\node_modules\_ali-rds@3.4.0@ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\12552\Desktop\dada\serve\node_modules\_co@4.6.0@co\index.js:65:19)
    at C:\Users\12552\Desktop\dada\serve\node_modules\_co@4.6.0@co\index.js:54:5
    sql: insert into 'order' (quality,uid,qianshou,price,time,info,sid)
         values ("undefined","undefined","undefined","undefined","undefined",
         "undefined"
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''order' (quality,uid,qianshou,price,time,info,sid)\n         values (\"undefined\",' at line 1"
sqlState: "42000"
index: 0
sql: "insert into 'order' (quality,uid,qianshou,price,time,info,sid)\n         values (\"undefined\",\"undefined\",\"undefined\",\"undefined\",\"undefined\",\n         \"undefined\""
headers: {"Access-Control-Allow-Origin":"http://192.168.3.150:3000","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_PARSE_ERRORError"
pid: 7944
hostname: DESKTOP-K5ULKD4

2020-10-16 15:59:09,700 ERROR 16800 [-/192.168.3.150/-/25ms GET /addorder?obj=%7B%22price%22:%22100~500%E5%85%83%22,%22quality%22:5,%22info%22:%22%E6%B5%8B%E8%AF%95%E6%95%B0%E6%8D%AE%22,%22qianshou%22:true,%22fahuoTime%22:%222020-10-21+15:49:23%22,%22address1%22:%22fgsg%22,%22address2%22:%22a%22,%22phone1%22:%2218212418574%22,%22phone2%22:%2218212418574%22,%22menpai1%22:%22d%22,%22menpai2%22:%2217%22,%22sid%22:1,%22uid%22:9%7D] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''order' (quality,uid,qianshou,price,time,info,sid)
         values ("undefined",' at line 1
         values ("undefined",' at line 1
    at Query.Sequence._packetToError (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Pool.js:199:23)
    at C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\12552\Desktop\dada\serve\node_modules\_ali-rds@3.4.0@ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (C:\Users\12552\Desktop\dada\serve\node_modules\_ali-rds@3.4.0@ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\12552\Desktop\dada\serve\node_modules\_co@4.6.0@co\index.js:65:19)
    at C:\Users\12552\Desktop\dada\serve\node_modules\_co@4.6.0@co\index.js:54:5
    sql: insert into 'order' (quality,uid,qianshou,price,time,info,sid)
         values ("undefined","undefined","undefined","undefined","undefined",
         "undefined")
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''order' (quality,uid,qianshou,price,time,info,sid)\n         values (\"undefined\",' at line 1"
sqlState: "42000"
index: 0
sql: "insert into 'order' (quality,uid,qianshou,price,time,info,sid)\n         values (\"undefined\",\"undefined\",\"undefined\",\"undefined\",\"undefined\",\n         \"undefined\")"
headers: {"Access-Control-Allow-Origin":"http://192.168.3.150:3000","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_PARSE_ERRORError"
pid: 16800
hostname: DESKTOP-K5ULKD4

2020-10-16 15:59:46,298 ERROR 17392 [-/192.168.3.150/-/18ms GET /addorder?obj=%7B%22price%22:%22100~500%E5%85%83%22,%22quality%22:5,%22info%22:%22%E6%B5%8B%E8%AF%95%E6%95%B0%E6%8D%AE%22,%22qianshou%22:true,%22fahuoTime%22:%222020-10-21+15:49:23%22,%22address1%22:%22fgsg%22,%22address2%22:%22a%22,%22phone1%22:%2218212418574%22,%22phone2%22:%2218212418574%22,%22menpai1%22:%22d%22,%22menpai2%22:%2217%22,%22sid%22:1,%22uid%22:9%7D] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''order' (quality,uid,qianshou,price,time,info,sid)
         values ("undefined",' at line 1
         values ("undefined",' at line 1
    at Query.Sequence._packetToError (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Pool.js:199:23)
    at C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\12552\Desktop\dada\serve\node_modules\_ali-rds@3.4.0@ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (C:\Users\12552\Desktop\dada\serve\node_modules\_ali-rds@3.4.0@ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\12552\Desktop\dada\serve\node_modules\_co@4.6.0@co\index.js:65:19)
    at C:\Users\12552\Desktop\dada\serve\node_modules\_co@4.6.0@co\index.js:54:5
    sql: insert into 'order' (quality,uid,qianshou,price,time,info,sid)
         values ("undefined","undefined","undefined","undefined","undefined",
         "undefined")
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''order' (quality,uid,qianshou,price,time,info,sid)\n         values (\"undefined\",' at line 1"
sqlState: "42000"
index: 0
sql: "insert into 'order' (quality,uid,qianshou,price,time,info,sid)\n         values (\"undefined\",\"undefined\",\"undefined\",\"undefined\",\"undefined\",\n         \"undefined\")"
headers: {"Access-Control-Allow-Origin":"http://192.168.3.150:3000","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_PARSE_ERRORError"
pid: 17392
hostname: DESKTOP-K5ULKD4

2020-10-16 16:00:59,799 ERROR 16380 [-/192.168.3.150/-/16ms GET /addorder?obj=%7B%22price%22:%22100~500%E5%85%83%22,%22quality%22:5,%22info%22:%22%E6%B5%8B%E8%AF%95%E6%95%B0%E6%8D%AE%22,%22qianshou%22:true,%22fahuoTime%22:%222020-10-21+15:49:23%22,%22address1%22:%22fgsg%22,%22address2%22:%22a%22,%22phone1%22:%2218212418574%22,%22phone2%22:%2218212418574%22,%22menpai1%22:%22d%22,%22menpai2%22:%2217%22,%22sid%22:1,%22uid%22:9%7D] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''order' (quality,uid,qianshou,price,time,info,sid)
         values ("undefined",' at line 1
         values ("undefined",' at line 1
    at Query.Sequence._packetToError (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Pool.js:199:23)
    at C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\12552\Desktop\dada\serve\node_modules\_ali-rds@3.4.0@ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (C:\Users\12552\Desktop\dada\serve\node_modules\_ali-rds@3.4.0@ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\12552\Desktop\dada\serve\node_modules\_co@4.6.0@co\index.js:65:19)
    at C:\Users\12552\Desktop\dada\serve\node_modules\_co@4.6.0@co\index.js:54:5
    sql: insert into 'order' (quality,uid,qianshou,price,time,info,sid)
         values ("undefined","undefined","undefined","undefined","undefined",
         "undefined")
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''order' (quality,uid,qianshou,price,time,info,sid)\n         values (\"undefined\",' at line 1"
sqlState: "42000"
index: 0
sql: "insert into 'order' (quality,uid,qianshou,price,time,info,sid)\n         values (\"undefined\",\"undefined\",\"undefined\",\"undefined\",\"undefined\",\n         \"undefined\")"
headers: {"Access-Control-Allow-Origin":"http://192.168.3.150:3000","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_PARSE_ERRORError"
pid: 16380
hostname: DESKTOP-K5ULKD4

2020-10-16 16:03:51,465 ERROR 12916 nodejs.ServerNoResponseError: server 127.0.0.1:55642 no response in 40000ms, maybe the socket is end on the other side. (address: 127.0.0.1:55642)
    at Timeout._onTimeout (C:\Users\12552\Desktop\dada\serve\node_modules\_tcp-base@3.1.0@tcp-base\lib\base.js:427:21)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ServerNoResponseError"
pid: 12916
hostname: DESKTOP-K5ULKD4

2020-10-16 16:04:53,899 ERROR 13520 nodejs.ClusterClientNoResponseError: client no response in 122433ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (C:\Users\12552\Desktop\dada\serve\node_modules\_cluster-client@3.0.1@cluster-client\lib\leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 13520
hostname: DESKTOP-K5ULKD4

2020-10-16 16:06:46,024 ERROR 11036 [-/192.168.3.150/-/26ms GET /addorder?obj=%7B%22price%22:%22100~500%E5%85%83%22,%22quality%22:5,%22info%22:%22%E6%B5%8B%E8%AF%95%E6%95%B0%E6%8D%AE%22,%22qianshou%22:true,%22fahuoTime%22:%222020-10-21+15:49:23%22,%22address1%22:%22fgsg%22,%22address2%22:%22a%22,%22phone1%22:%2218212418574%22,%22phone2%22:%2218212418574%22,%22menpai1%22:%22d%22,%22menpai2%22:%2217%22,%22sid%22:1,%22uid%22:9%7D] nodejs.ER_WRONG_VALUE_COUNT_ON_ROWError: ER_WRONG_VALUE_COUNT_ON_ROW: Column count doesn't match value count at row 1
    at Query.Sequence._packetToError (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Pool.js:199:23)
    at C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\12552\Desktop\dada\serve\node_modules\_ali-rds@3.4.0@ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (C:\Users\12552\Desktop\dada\serve\node_modules\_ali-rds@3.4.0@ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\12552\Desktop\dada\serve\node_modules\_co@4.6.0@co\index.js:65:19)
    at C:\Users\12552\Desktop\dada\serve\node_modules\_co@4.6.0@co\index.js:54:5
    sql: insert into `order` (quality,uid,qianshou,price,time,info,sid)
         values ("undefined","undefined","undefined","undefined","undefined",
         "undefined")
code: "ER_WRONG_VALUE_COUNT_ON_ROW"
errno: 1136
sqlMessage: "Column count doesn't match value count at row 1"
sqlState: "21S01"
index: 0
sql: "insert into `order` (quality,uid,qianshou,price,time,info,sid)\n         values (\"undefined\",\"undefined\",\"undefined\",\"undefined\",\"undefined\",\n         \"undefined\")"
headers: {"Access-Control-Allow-Origin":"http://192.168.3.150:3000","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_WRONG_VALUE_COUNT_ON_ROWError"
pid: 11036
hostname: DESKTOP-K5ULKD4

2020-10-16 16:06:58,386 ERROR 11036 [-/192.168.3.150/-/12ms GET /addorder?obj=%7B%22price%22:%22100~500%E5%85%83%22,%22quality%22:5,%22info%22:%22%E6%B5%8B%E8%AF%95%E6%95%B0%E6%8D%AE%22,%22qianshou%22:true,%22fahuoTime%22:%222020-10-21+15:49:23%22,%22address1%22:%22fgsg%22,%22address2%22:%22a%22,%22phone1%22:%2218212418574%22,%22phone2%22:%2218212418574%22,%22menpai1%22:%22d%22,%22menpai2%22:%2217%22,%22sid%22:1,%22uid%22:9%7D] nodejs.ER_WRONG_VALUE_COUNT_ON_ROWError: ER_WRONG_VALUE_COUNT_ON_ROW: Column count doesn't match value count at row 1
    at Query.Sequence._packetToError (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Pool.js:199:23)
    at C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\12552\Desktop\dada\serve\node_modules\_ali-rds@3.4.0@ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (C:\Users\12552\Desktop\dada\serve\node_modules\_ali-rds@3.4.0@ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\12552\Desktop\dada\serve\node_modules\_co@4.6.0@co\index.js:65:19)
    at C:\Users\12552\Desktop\dada\serve\node_modules\_co@4.6.0@co\index.js:54:5
    sql: insert into `order` (quality,uid,qianshou,price,time,info,sid)
         values ("undefined","undefined","undefined","undefined","undefined",
         "undefined")
code: "ER_WRONG_VALUE_COUNT_ON_ROW"
errno: 1136
sqlMessage: "Column count doesn't match value count at row 1"
sqlState: "21S01"
index: 0
sql: "insert into `order` (quality,uid,qianshou,price,time,info,sid)\n         values (\"undefined\",\"undefined\",\"undefined\",\"undefined\",\"undefined\",\n         \"undefined\")"
headers: {"Access-Control-Allow-Origin":"http://192.168.3.150:3000","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_WRONG_VALUE_COUNT_ON_ROWError"
pid: 11036
hostname: DESKTOP-K5ULKD4

2020-10-16 16:11:59,405 ERROR 17120 [-/192.168.3.150/-/13ms GET /addorder?obj=%7B%22price%22:%22100~500%E5%85%83%22,%22quality%22:5,%22info%22:%22%E6%B5%8B%E8%AF%95%E6%95%B0%E6%8D%AE%22,%22qianshou%22:true,%22fahuoTime%22:%222020-10-21+15:49:23%22,%22address1%22:%22fgsg%22,%22address2%22:%22a%22,%22phone1%22:%2218212418574%22,%22phone2%22:%2218212418574%22,%22menpai1%22:%22d%22,%22menpai2%22:%2217%22,%22sid%22:1,%22uid%22:9%7D] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'order (quality,uid,qianshou,price,time,info,sid) values ("undefined","undefined"' at line 1
    at Query.Sequence._packetToError (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Pool.js:199:23)
    at C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\12552\Desktop\dada\serve\node_modules\_ali-rds@3.4.0@ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (C:\Users\12552\Desktop\dada\serve\node_modules\_ali-rds@3.4.0@ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\12552\Desktop\dada\serve\node_modules\_co@4.6.0@co\index.js:65:19)
    at C:\Users\12552\Desktop\dada\serve\node_modules\_co@4.6.0@co\index.js:54:5
    sql: insert into order (quality,uid,qianshou,price,time,info,sid) values ("undefined","undefined","undefined","undefined","undefined",
         "undefined")
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'order (quality,uid,qianshou,price,time,info,sid) values (\"undefined\",\"undefined\"' at line 1"
sqlState: "42000"
index: 0
sql: "insert into order (quality,uid,qianshou,price,time,info,sid) values (\"undefined\",\"undefined\",\"undefined\",\"undefined\",\"undefined\",\n         \"undefined\")"
headers: {"Access-Control-Allow-Origin":"http://192.168.3.150:3000","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_PARSE_ERRORError"
pid: 17120
hostname: DESKTOP-K5ULKD4

2020-10-16 16:17:23,230 ERROR 7488 [-/192.168.3.150/-/12ms GET /addorder?obj=%7B%22price%22:%22100~500%E5%85%83%22,%22quality%22:5,%22info%22:%22%E6%B5%8B%E8%AF%95%E6%95%B0%E6%8D%AE%22,%22qianshou%22:true,%22fahuoTime%22:%222020-10-21+15:49:23%22,%22address1%22:%22fgsg%22,%22address2%22:%22a%22,%22phone1%22:%2218212418574%22,%22phone2%22:%2218212418574%22,%22menpai1%22:%22d%22,%22menpai2%22:%2217%22,%22sid%22:1,%22uid%22:9%7D] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'order (quality,uid,qianshou,price,time,info,sid) values ("undefined","undefined"' at line 1
    at Query.Sequence._packetToError (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Pool.js:199:23)
    at C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\12552\Desktop\dada\serve\node_modules\_ali-rds@3.4.0@ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (C:\Users\12552\Desktop\dada\serve\node_modules\_ali-rds@3.4.0@ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\12552\Desktop\dada\serve\node_modules\_co@4.6.0@co\index.js:65:19)
    at C:\Users\12552\Desktop\dada\serve\node_modules\_co@4.6.0@co\index.js:54:5
    sql: insert into order (quality,uid,qianshou,price,time,info,sid) values ("undefined","undefined","undefined","undefined","undefined",
         "undefined")
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'order (quality,uid,qianshou,price,time,info,sid) values (\"undefined\",\"undefined\"' at line 1"
sqlState: "42000"
index: 0
sql: "insert into order (quality,uid,qianshou,price,time,info,sid) values (\"undefined\",\"undefined\",\"undefined\",\"undefined\",\"undefined\",\n         \"undefined\")"
headers: {"Access-Control-Allow-Origin":"http://192.168.3.150:3000","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_PARSE_ERRORError"
pid: 7488
hostname: DESKTOP-K5ULKD4

2020-10-16 16:20:55,143 ERROR 4364 [-/192.168.3.150/-/17ms GET /addorder?obj=%7B%22price%22:%22100~500%E5%85%83%22,%22quality%22:5,%22info%22:%22%E6%B5%8B%E8%AF%95%E6%95%B0%E6%8D%AE%22,%22qianshou%22:true,%22fahuoTime%22:%222020-10-21+15:49:23%22,%22address1%22:%22fgsg%22,%22address2%22:%22a%22,%22phone1%22:%2218212418574%22,%22phone2%22:%2218212418574%22,%22menpai1%22:%22d%22,%22menpai2%22:%2217%22,%22sid%22:1,%22uid%22:9%7D] nodejs.ER_WRONG_VALUE_COUNT_ON_ROWError: ER_WRONG_VALUE_COUNT_ON_ROW: Column count doesn't match value count at row 1
    at Query.Sequence._packetToError (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Pool.js:199:23)
    at C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\12552\Desktop\dada\serve\node_modules\_ali-rds@3.4.0@ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (C:\Users\12552\Desktop\dada\serve\node_modules\_ali-rds@3.4.0@ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\12552\Desktop\dada\serve\node_modules\_co@4.6.0@co\index.js:65:19)
    at C:\Users\12552\Desktop\dada\serve\node_modules\_co@4.6.0@co\index.js:54:5
    sql: insert into myorder (quality,uid,qianshou,price,time,info,sid) values ("undefined","undefined","undefined","undefined","undefined",
         "undefined")
code: "ER_WRONG_VALUE_COUNT_ON_ROW"
errno: 1136
sqlMessage: "Column count doesn't match value count at row 1"
sqlState: "21S01"
index: 0
sql: "insert into myorder (quality,uid,qianshou,price,time,info,sid) values (\"undefined\",\"undefined\",\"undefined\",\"undefined\",\"undefined\",\n         \"undefined\")"
headers: {"Access-Control-Allow-Origin":"http://192.168.3.150:3000","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_WRONG_VALUE_COUNT_ON_ROWError"
pid: 4364
hostname: DESKTOP-K5ULKD4

2020-10-16 16:38:40,459 ERROR 1188 [-/192.168.3.150/-/17ms GET /addorder?price=100~500%E5%85%83&quality=16&info=&qianshou=true&fahuoTime=2020-10-16T08:32:59.552Z&address1=dasd&address2=1&phone1=18212418574&phone2=18212418574&menpai1=asda&menpai2=s&sid=1&uid=9] nodejs.ER_WRONG_VALUE_COUNT_ON_ROWError: ER_WRONG_VALUE_COUNT_ON_ROW: Column count doesn't match value count at row 1
    at Query.Sequence._packetToError (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Pool.js:199:23)
    at C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\12552\Desktop\dada\serve\node_modules\_ali-rds@3.4.0@ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (C:\Users\12552\Desktop\dada\serve\node_modules\_ali-rds@3.4.0@ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\12552\Desktop\dada\serve\node_modules\_co@4.6.0@co\index.js:65:19)
    at C:\Users\12552\Desktop\dada\serve\node_modules\_co@4.6.0@co\index.js:54:5
    sql: insert into myorder (quality,uid) values ("16","9","true","100~500元","2020-10-16T08:32:59.552Z",
         "1")
code: "ER_WRONG_VALUE_COUNT_ON_ROW"
errno: 1136
sqlMessage: "Column count doesn't match value count at row 1"
sqlState: "21S01"
index: 0
sql: "insert into myorder (quality,uid) values (\"16\",\"9\",\"true\",\"100~500元\",\"2020-10-16T08:32:59.552Z\",\n         \"1\")"
headers: {"Access-Control-Allow-Origin":"http://192.168.3.150:3000","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_WRONG_VALUE_COUNT_ON_ROWError"
pid: 1188
hostname: DESKTOP-K5ULKD4

2020-10-16 16:41:23,636 ERROR 2708 [-/192.168.3.150/-/22ms GET /addorder?price=100~500%E5%85%83&quality=16&info=&qianshou=true&fahuoTime=2020-10-16T08:32:59.552Z&address1=dasd&address2=1&phone1=18212418574&phone2=18212418574&menpai1=asda&menpai2=s&sid=1&uid=9] nodejs.ER_TRUNCATED_WRONG_VALUE_FOR_FIELDError: ER_TRUNCATED_WRONG_VALUE_FOR_FIELD: Incorrect integer value: '' for column 'sid' at row 1
    at Query.Sequence._packetToError (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    --------------------
    at Pool.query (C:\Users\12552\Desktop\dada\serve\node_modules\_mysql@2.18.1@mysql\lib\Pool.js:199:23)
    at C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\12552\Desktop\dada\serve\node_modules\_pify@2.3.0@pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\12552\Desktop\dada\serve\node_modules\_ali-rds@3.4.0@ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (C:\Users\12552\Desktop\dada\serve\node_modules\_ali-rds@3.4.0@ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\12552\Desktop\dada\serve\node_modules\_co@4.6.0@co\index.js:65:19)
    at C:\Users\12552\Desktop\dada\serve\node_modules\_co@4.6.0@co\index.js:54:5
    sql: insert into myorder (quality,uid,qianshou,price,time,info,sid) values ("16","9","true","100~500元","2020-10-16T08:32:59.552Z",
         "1","")
code: "ER_TRUNCATED_WRONG_VALUE_FOR_FIELD"
errno: 1366
sqlMessage: "Incorrect integer value: '' for column 'sid' at row 1"
sqlState: "HY000"
index: 0
sql: "insert into myorder (quality,uid,qianshou,price,time,info,sid) values (\"16\",\"9\",\"true\",\"100~500元\",\"2020-10-16T08:32:59.552Z\",\n         \"1\",\"\")"
headers: {"Access-Control-Allow-Origin":"http://192.168.3.150:3000","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ER_TRUNCATED_WRONG_VALUE_FOR_FIELDError"
pid: 2708
hostname: DESKTOP-K5ULKD4

